samples/word/50-document/insert-external-document.yaml

```diff
@@ -0,0 +1,101 @@
+order: 2
+id: word-document-insert-external-document
+name: Insert an external document
+description: Inserts an external document into the current document.
+host: WORD
+api_set:
+    WordApi: 1.1
+script:
+    content: |
+        $("#file").change(getBase64);
+        $("#insert-document").click(() => tryCatch(insertDocument));
+
+        let externalDocument;
+
+        async function insertDocument() {
+          // Update the text of the current document with the text from another document passed in as a Base64-encoded string.
+          await Word.run(async (context) => {
+            // Use the Base64-encoded string representation of the selected .docx file.
+            const externalDoc = context.application.createDocument(externalDocument);
+            await context.sync();
+
+            const externalDocBody = externalDoc.body;
+            externalDocBody.load("text");
+            await context.sync();
+
+            const externalDocBodyText = externalDocBody.text;
```

Comment:
Consider adding a comment like "Add the other document's text to the beginning of the current document's body."